cfpb/cfgov-refresh
# Paying for College: Add program codes to API

Now that we have program-level data from College Scorecard, we can
provide a detailed list of available programs to the financial-path tool
so that students can find their specific degree.

## How to test this PR

With a recent production database dump, check out the branch and check the
school API endpoint for George Washington University.

<http://localhost:8000/paying-for-college2/understanding-your-financial-aid-offer/api/school/131469/>

You should see a new value called programCodes with entries like this:

```python
{
    "graduate": [
        {
            "code": "501-5",
            "name": "Area Studies",
            "level": "Master's degree",
            "salary": 51700,
        },
        {
            "code": "909-5",
            "name": "Public Relations, Advertising, and Applied Communication",
            "level": "Master's degree",
            "salary": 61800,
        }
    ]
    "undergrad": []
}
```

## Note

We have only fetched this detailed data for GWU.

[GHE]/paying-for-college/dynamic-disclosures-documentation/wiki/API-documentation#json-result-1
